src/ansible_compat/constants.py

```diff
@@ -33,8 +33,8 @@
 role_name: my_name  # if absent directory name hosting role is used instead
 namespace: my_galaxy_namespace  # if absent, author is used instead
 
-Namespace: https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespace-limitations
-Role: https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names
+Namespace: https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/usage_guide/collections/#permissions
```

Comment:
@tanwigeetika1618 please update the link to point to [namespace](https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/usage_guide/collections/#namespaces) instead of permissions

Reply:
Revised the link in the namespaces section of constants.py . The updated link now points to the most accurate and current resource for namespaces.

Reply:
Looking at where `Namespace` and `Role` links (https://old-galaxy.ansible.com/docs/contributing/creating_role.html#role-names, https://old-galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespace-limitations respectively) have pointed previously, I believe the intent was to provide documentation on what constitutes a valid namespace and role name. Are we sure these new links convey the same thing?